lib/plugins/blocks.js

```diff
@@ -268,6 +268,24 @@ function inject (bot, { version, storageBuilder, hideErrors }) {
     }
   })
 
+  // Chunk batches are used by the server to throttle the chunks per tick for players based on their connection speed.
+  let chunkBatchStartTime = 0
+  let weightedAverage = 2 // The Vanilla client uses nano seconds with its weighted average starting at 2000000 converted to milliseconds that is 2
+  let oldSampleWeight = 1 // This is used for keeping track of the weight of the old average when updating it.
+
+  bot._client.on('chunk_batch_start', (packet) => {
+    chunkBatchStartTime = Date.now() // Get the time the chunk batch is starting.
+  })
+
+  bot._client.on('chunk_batch_finished', (packet) => {
+    const MilliPerChunk = (Date.now() - chunkBatchStartTime) / packet.batchSize // Gets millisecond per chunk
```

Comment:
Should start variable named with lower case

Reply:
I will fix that when I get home, surprised the linter didnt carch that.

Reply:
Both now start with lower case